operator/pkg/apis/istio/v1alpha1/validation/validation.go

```diff
@@ -70,13 +76,9 @@ func firstCharsToLower(s string) string {
 }
 
 func deprecatedSettingsMessage(iop *v1alpha1.IstioOperatorSpec) string {
+	const mixerDeprecatedMessage = "! Using deprecated setting: %s. Mixer is deprecated and will be removed from Istio with the 1.8 release.\n Please consult our docs on the replacement."
```

Comment:
very nitpick: now that this is only used in one place inline it or define it right before we use it for clarity

Reply:
```suggestion
	const mixerDeprecatedMessage = "! %s is deprecated; Mixer is deprecated and will be removed from Istio with the 1.8 release. Please consult our docs on the replacement."
```

for consistency

Reply:
done